### PR TITLE
Fix enum migration for Postgres

### PR DESCRIPTION
**Pull request by supergillis**
Fixes enum migration in issue #137

### CODE REVIEW

File: database/migrations/2017_06_07_173437_add_multiple_genders_choices.php [hunk 1]
```diff
@@ -13,8 +13,40 @@ class AddMultipleGendersChoices extends Migration
...
+                DB::statement("ALTER TABLE significant_others CHANGE COLUMN gender gender ENUM('male', 'female', 'none')");
+                DB::statement("ALTER TABLE kids CHANGE COLUMN gender gender ENUM('male', 'female', 'none')");
+                break;
+            case 'pgsql':
+                $this->alterEnum('contacts', 'gender', ['male', 'female', 'none']);
+                $this->alterEnum('significant_others', 'gender', ['male', 'female', 'none']);
+                $this->alterEnum('kids', 'gender', ['male', 'female', 'none']);
+                break;
+            default:
+                throw new \Exception('Driver not supported.');
```
Comment (Phrohdoh) [inline database/migrations/2017_06_07_173437_add_multiple_genders_choices.php]: Would you put the driver name in this string, please?

It is easier to help someone with a message `Driver foobar is not supported`.

Comment (djaiss) [inline database/migrations/2017_06_07_173437_add_multiple_genders_choices.php]: Good point.

Comment (supergillis) [inline database/migrations/2017_06_07_173437_add_multiple_genders_choices.php]: Done.

**Comment by djaiss** [comment]
Thanks a lot for this PR!

**Pull request merged**